atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SweepCommand.java

```diff
@@ -132,11 +136,17 @@ public int execute(final AtlasDbServices services) {
 
             while (startRow.isPresent()) {
                 Stopwatch watch = Stopwatch.createStarted();
-                SweepResults results = sweepRunner.run(table, batchSize, cellBatchSize, startRow.get());
-                printer.info("Swept from {} to {} in table {} in {} ms, examined {} unique cells, deleted {} stale versions of those cells.",
+
+                SweepResults results = dryRun
+                        ? sweepRunner.dryRun(table, batchSize, cellBatchSize, startRow.get())
+                        : sweepRunner.run(table, batchSize, cellBatchSize, startRow.get());
+                printer.info(
+                        "Swept from {} to {} in table {} in {} ms, examined {} unique cells, {}deleted {} stale versions of those cells.",
                         encodeStartRow(startRow), encodeEndRow(results.getNextStartRow()),
                         table, watch.elapsed(TimeUnit.MILLISECONDS),
-                        results.getCellsExamined(), results.getCellsDeleted());
+                        results.getCellsExamined(),
+                        dryRun ? "would have " : "",
+                        results.getCellsDeleted());
                 startRow = results.getNextStartRow();
```

Comment:
As discussed with @gsheasby we need to be careful here that dry run doesn't result in modifications to the sweep priority table.  Currently a dry run would update the sweep priority table which would be rather unfortunate.  Beyond this it would be good to have a test using a spy to ensure the only puts / deletes are actioned on the sweep progress table.  Should be doable with the cli test runner set up.